attribute-service-api/src/main/proto/org/hypertrace/core/attribute/service/v1/attribute_metadata.proto

```diff
@@ -104,6 +104,7 @@ message AttributeMetadata {
   bool internal = 18;
   bool custom = 19;
   string description = 20;
+  string data_extraction_path = 21;
```

Comment:
This already exists, it's part of the attribute definition

Reply:
Oh i see then we dont need this PR itself..thanks